src/py/flwr/server/serverapp/app.py

```diff
@@ -124,47 +93,29 @@ def flwr_serverapp() -> None:
     restore_output()
 
 
-def _try_obtain_certificates(
-    args: argparse.Namespace,
-) -> Optional[bytes]:
-
-    if args.insecure:
-        if args.root_certificates is not None:
-            sys.exit(
-                "Conflicting options: The '--insecure' flag disables HTTPS, "
-                "but '--root-certificates' was also specified. Please remove "
-                "the '--root-certificates' option when running in insecure mode, "
-                "or omit '--insecure' to use HTTPS."
-            )
-        log(
-            WARN,
-            "Option `--insecure` was set. Starting insecure HTTP channel to %s.",
-            args.superlink,
-        )
-        root_certificates = None
-    else:
-        # Load the certificates if provided, or load the system certificates
-        if not isfile(args.root_certificates):
-            sys.exit("Path argument `--root-certificates` does not point to a file.")
-        root_certificates = Path(args.root_certificates).read_bytes()
-        log(
-            DEBUG,
-            "Starting secure HTTPS channel to %s "
-            "with the following certificates: %s.",
-            args.superlink,
-            args.root_certificates,
-        )
-    return root_certificates
-
-
 def run_serverapp(  # pylint: disable=R0914, disable=W0212
     superlink: str,
     log_queue: Queue[Optional[str]],
     run_once: bool,
     flwr_dir_: Optional[str] = None,
     certificates: Optional[bytes] = None,
 ) -> None:
-    """Run Flower ServerApp process."""
+    """Run Flower ServerApp process.
+
+    Parameters
+    ----------
+    superlink: str
+        Address of SuperLink's ServerAppIo gRPC servicer
+    log_queue: Queue[Optional[str]]
+        Queue to capture logs
+    run_once: bool
+        Boolean flag to run the ServerApp process only once for a single run
+    flwr_dir_: Optional[str] (default=None)
+        Directory where FABs are installed
+    certificates: Optional[bytes] (default=None)
+        Root certificates for establishing secure HTTPS connection between SuperLink
+        and `flwr-serverapp` process
+    """
```

Comment:
do we need this? this function shouldn't be called by anything. 

@panh99, thoughts?

Reply:
if we decide not to include the above, let's remove them also from `flwr-clientapp`?

Reply:
@jafermarq @panh99 IMO, this would be helpful for admins who need to spin up `flwr-*app` containers in `--isolation process` mode.

Reply:
but for that don't they see the argparse help?